vars/main.yml

```diff
@@ -54,3 +54,5 @@ php_cli_ondrej_mods_dependencies:
     - php-ast
   sqlite3:
     - "php{{ php_cli_ondrej_version }}-sqlite3"
+  libsodium:
+    - "php{{ php_cli_ondrej_version }}-libsodium"
```

Comment:
There is but there is also a symlink (internally) to the version without the version number.
I tought in case of a change in the future, we use the versioned version ;)